specification/validated-relationships-service-api.yaml

```diff
@@ -377,22 +546,29 @@ components:
                               properties:
                                 system:
                                   type: string
-                                  enum: ['http://terminology.hl7.org/CodeSystem/v3-RoleCode']
+                                  enum:
+                                    [
+                                      "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+                                      "http://terminology.hl7.org/CodeSystem/v3-RoleClass",
+                                      "https://fhir.hl7.org.uk/CodeSystem/UKCore-AdditionalPersonRelationshipRole"
+                                    ]                                
                                 code:
                                   type: string
-                                  enum: ['PRN', 'Personal']
+                                  enum: ['PRN', 'CHILD', 'SPS', 'FAMMEMB', 'FRND', 'CAREGIVER', 'SIGOTHR']
                                 display:
                                   type: string
-                                  example: "Personal"
-                                  
-    QuestionnaireItem_PatientDetails:
+                                  description: FHIR relationship type.
+                                  enum: ['Parent', 'Child', 'Partner, wife or husband', 'Other family member', 'Friend, neighbour or housemate', 'Paid carer or support worker', 'Other']
+
+    QuestionnaireResponseItem_ParentalApplicationSupplementaryDetails:
```

Comment:
Think we're missing three of the child properties being required here (to align with the Questionnaire) https://nhsdigital.github.io/interoperability-standards-tools-skunkworks/questionnaire?url=https:%2F%2Ffhir.nhs.uk%2FEngland%2FQuestionnaire%2FProxyAccessRequest

- Evidence
- Address confirmed 
- do they live at the same address.

Reply:
These should be found under:

- parentalApplicationSupplementaryDetails_evidenceOfResponsibility
- parentalApplicationSupplementaryDetails_isCurrentAddressConfirmed
- parentalApplicationSupplementaryDetails_liveAtSameAddress

Reply:
Clarified with @miiisterjim 
All of these elements need to have required for link, text and answer

Updated as such